Formula/halibut.rb

```diff
@@ -1,9 +1,10 @@
 class Halibut < Formula
   desc "Yet another free document preparation system"
   homepage "https://www.chiark.greenend.org.uk/~sgtatham/halibut/"
-  url "https://www.chiark.greenend.org.uk/~sgtatham/halibut/halibut-1.2/halibut-1.2.tar.gz"
-  sha256 "1aedfb6240f27190c36a390fcac9ce732edbdbaa31c85ee675b994e2b083163f"
-  head "https://git.tartarus.org/simon/halibut.git"
+  url "https://www.chiark.greenend.org.uk/~sgtatham/halibut/halibut-1.3/halibut-1.3.tar.gz"
+  sha256 "aaa0f7696f17f74f42d97d0880aa088f5d68ed3079f3ed15d13b6e74909d3132"
+  license "MIT"
```

Comment:
```suggestion
  license :cannot_represent
```

Reply:
Probably more like
```ruby
license all_of: ["MIT", :cannot_represent]
```
Or whatever that second license is.